tests/contrib/io/azure/test_csv_blob.py

```diff
@@ -62,6 +74,103 @@ def make_data_set(load_args=None, save_args=None):
     return make_data_set
 
 
+@pytest.fixture
+def versioned_blob_csv_data_set(load_version, save_version):
+    return CSVBlobDataSet(
+        filepath=TEST_FILE_NAME,
+        container_name=TEST_CONTAINER_NAME,
+        credentials=TEST_CREDENTIALS,
+        blob_to_text_args={"to_extra": 42},
+        blob_from_text_args={"from_extra": 42},
+        version=Version(load_version, save_version),
+    )
+
+
+@pytest.fixture
+def save_path(save_version):
+    return "{0}/{1}/{0}".format(TEST_FILE_NAME, save_version)
+
+
+class TestCSVBlobDataSetVersioned:
+    # pylint: disable=too-many-arguments
+    @patch("kedro.contrib.io.azure.csv_blob.BlockBlobService.create_blob_from_text")
+    @patch(
+        "kedro.contrib.io.azure.csv_blob.BlockBlobService.exists", return_value=False
+    )
+    def test_save(
```

Comment:
And could you please unit test for `load()` as well?

Reply:
Done!